backend/service/k8s/clientset.go

```diff
@@ -138,12 +138,12 @@ type managerImpl struct {
 	clientsets map[string]*ctxClientsetImpl
 }
 
-func (m *managerImpl) Clientsets(ctx context.Context) map[string]ContextClientset {
+func (m *managerImpl) Clientsets(ctx context.Context) (map[string]ContextClientset, error) {
```

Comment:
in this func, not seeing a case when an error is returned

Reply:
it would only happen internally currently where clientsets are dynamic